wordpress-mobile/gutenberg-mobile
# Update post title padding

This PR adds some vertical padding to post title.

gutenberg PR https://github.com/WordPress/gutenberg/pull/14040

To Test:

- Run metro on this branch
- Open a post and verify post title has top&bottom paddings
